### PR TITLE
Timestamp

### PR DESCRIPTION
**Pull request by BabyCakes13**
Resolves issue #6. 

Besides that, it also adds more items to sensor collection. We should check what else is needed / not necessary.

- type: type of the sensor (aka light, motion, environment...);
- name: name of the sensor (TSL258x...);
- value: value collected by the sensor for the specific type;
- **timestamp**
- measurement: measurement unit for the specific collected value.

### CODE REVIEW

File: homesensorhub/communication/data_sender.py [hunk 1]
```diff
@@ -48,9 +48,7 @@ def send_data(self, data):
 
     def send_current(self, value, topic):
         topic = self.HOST + "/" + topic + "/current"
-        payload = value #TODO: This give us invalid syntax: f'{value:.3f}'
-        logging.info("Sending data to :" + topic + " --> " + str(payload))
-
+        payload = str(value)  # TODO: This give us invalid syntax: f'{value:.3f}
```
Comment (alex-sever-h) [inline homesensorhub/communication/data_sender.py:51]: This should be converted to a limited number of decimal points to be easier to interpret on the other side.

File: homesensorhub/sensors/environmental_sensor.py [hunk 2]
```diff
@@ -39,11 +40,26 @@ def collect_data(self) -> dict:
...
+                                                    'celsius'
+                                                    ),
+            'humidity': self.build_sensor_packet('humidity',
+                                                 sensor.humidity,
+                                                 datetime.datetime.now(),
+                                                 'percent'),
+            'pressure': self.build_sensor_packet('pressure',
+                                                 sensor.pressure,
+                                                 datetime.datetime.now(),
+                                                 'UF/USA'),
```
Comment (alex-sever-h) [inline homesensorhub/sensors/environmental_sensor.py:58]: what does UF/USA mean ?

File: homesensorhub/sensors/environmental_sensor.py [hunk 3]
```diff
@@ -39,11 +40,26 @@ def collect_data(self) -> dict:
...
+            'altitude': self.build_sensor_packet('altitude',
+                                                 sensor.altitude,
+                                                 datetime.datetime.now(),
+                                                 'not sure'),
```
Comment (alex-sever-h) [inline homesensorhub/sensors/environmental_sensor.py:62]: Altitude is measured in meters. :)
Also, might be better to calculate the other way around: specify altitude as a setting for the sensor, and calculate pressure at sea-level, not just local pressure, to be able to compare with weather station values.

File: homesensorhub/sensors/sensor.py [hunk 4]
```diff
@@ -14,6 +14,20 @@ def __init__(self, sensor, name):
         self.__sensor = sensor
         self.__name = name
 
+    def build_sensor_packet(self, type, value, timestamp, measurement):
+        """Build a packet with information taken from each sensor."""
+        print(self.__name)
```
Comment (alex-sever-h) [inline homesensorhub/sensors/sensor.py:19]: remove debug print

File: homesensorhub/sensors/environmental_sensor.py (hunk 2 above)
Comment (BabyCakes13) [reply homesensorhub/sensors/environmental_sensor.py:58]: Units of Force / Units of squared Area. It was supposed to be the measurement for the pressure, but after checking the adafruit bme280 library, it is measured in hectoPascals (I will change it):

 @property
    def pressure(self):
        """
        The compensated pressure in hectoPascals.
        returns None if pressure measurement is disabled
        """
https://github.com/adafruit/Adafruit_CircuitPython_BME280/blob/master/adafruit_bme280.py

File: homesensorhub/sensors/sensor.py (hunk 4 above)
Comment (BabyCakes13) [reply homesensorhub/sensors/sensor.py:19]: Solved in 92d88406c2998f664c3f53a9cca80e3f51d92814.

File: homesensorhub/sensors/environmental_sensor.py (hunk 3 above)
Comment (BabyCakes13) [reply homesensorhub/sensors/environmental_sensor.py:62]: I have created #9 to fix this. For now I will change the unit of measurement.

File: homesensorhub/communication/data_sender.py (hunk 1 above)
Comment (BabyCakes13) [reply homesensorhub/communication/data_sender.py:51]: You mean the actual value read by the sensor, right? The payload is of the form:

{'type': 'temperature', 'name': <class 'adafruit_bme280.Adafruit_BME280_I2C'>, 'value': 29.604296875, 'timestamp': datetime.datetime(2020, 8, 1, 15, 26, 9, 29915), 'measurement': 'celsius'}.

Comment (BabyCakes13) [reply homesensorhub/communication/data_sender.py:51]: I will include this in #10.

File: homesensorhub/sensors/environmental_sensor.py (hunk 3 above)
Comment (BabyCakes13) [reply homesensorhub/sensors/environmental_sensor.py:62]: I will include this fix in #9.

**Pull request merged**